test/format/format.sh

```diff
@@ -199,15 +204,53 @@ success=0
 running=0
 status="format.sh-status"
 
+skip_known_errors()
```

Comment:
It would be nice to add a function comment to explain the meaning of return values. I guess 0 means we find a known error and skip it, 1 means skip_errors flag not set or no (known error) match found?

Reply:
Fixed